README.md

```diff
@@ -1,7 +1,116 @@
 python-gp
 ===========
+[![Python3.6](https://img.shields.io/badge/python-3.6-red.svg)](https://www.python.org/downloads/release/python-369/)
+[![Python3.7](https://img.shields.io/badge/python-3.7-red.svg)](https://www.python.org/)
+[![Python3.8](https://img.shields.io/badge/python-3.8-red.svg)](https://www.python.org/)
+[![GPL license](https://img.shields.io/badge/License-GPLv3-blue.svg)](https://www.gnu.org/licenses/old-licenses/gpl-3.0.html)
+
+Cartesian Genetic Programming (CGP) in Python.
+
+This library implements Cartesian Genetic Programming in pure Python. It provides Python data structures to represent and evolve a two-dimensional directed graph (the genotype) that can be translated into a computational graph (the phenotype) implementing a particular mathematical expression. The current implementation uses an evolutionary algorithm,  (mu + lambda) evolution strategies, to evolve a population of Cartesian graphs in order to optimize an objective function.
```

Comment:
```suggestion
This library implements Cartesian Genetic Programming for symbolic regression in pure Python. It provides Python data structures to represent and evolve a two-dimensional directed graph (the genotype) that can be translated into a computational graph (the phenotype) implementing a particular mathematical expression. The current implementation uses an evolutionary algorithm,  specifically (mu + lambda) evolution strategies, to evolve a population of symbolic expressions in order to optimize an objective function.

```

Reply:
in the manuscript we refer to the flattened 1d array of integers as the genotype, here we refer to the 2d directed graph as the genotype. i think we should settle on one; i'd prefer the latter and adapt the manuscript accordingly. the flattened array of ints is an implementation detail imho

Reply:
Done.